[2022-02-15 02:28:53,292] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.stg_orders manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:28:53,328] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.stg_orders manual__2022-02-15T02:27:57.149490+00:00 [queued]>
[2022-02-15 02:28:53,329] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:28:53,329] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 02:28:53,330] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 02:28:53,378] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): test.my_new_project.stg_orders> on 2022-02-15 02:27:57.149490+00:00
[2022-02-15 02:28:53,394] {standard_task_runner.py:52} INFO - Started process 798 to run task
[2022-02-15 02:28:53,403] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'test.my_new_project.stg_orders', 'manual__2022-02-15T02:27:57.149490+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmpm8u4vcn3', '--error-file', '/tmp/tmpv72za2js']
[2022-02-15 02:28:53,405] {standard_task_runner.py:77} INFO - Job 75: Subtask test.my_new_project.stg_orders
[2022-02-15 02:28:53,696] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.test.my_new_project.stg_orders manual__2022-02-15T02:27:57.149490+00:00 [running]> on host de7a240d0d46
[2022-02-15 02:28:53,918] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=test.my_new_project.stg_orders
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T02:27:57.149490+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T02:27:57.149490+00:00
[2022-02-15 02:28:53,936] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 02:28:53,943] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json test --target dev --models stg_orders             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 02:28:54,074] {subprocess.py:85} INFO - Output:
[2022-02-15 02:29:07,459] {subprocess.py:89} INFO - 02:29:07  Running with dbt=1.0.1
[2022-02-15 02:29:07,877] {subprocess.py:89} INFO - 02:29:07  [WARNING]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2022-02-15 02:29:07,882] {subprocess.py:89} INFO - There are 2 unused configuration paths:
[2022-02-15 02:29:07,886] {subprocess.py:89} INFO - - models.jaffle_shop.marts.core
[2022-02-15 02:29:07,887] {subprocess.py:89} INFO - - models.jaffle_shop.staging
[2022-02-15 02:29:07,888] {subprocess.py:89} INFO - 
[2022-02-15 02:29:07,905] {subprocess.py:89} INFO - 02:29:07  Found 7 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 02:29:07,913] {subprocess.py:89} INFO - 02:29:07
[2022-02-15 02:29:10,733] {subprocess.py:89} INFO - 02:29:10  Concurrency: 4 threads (target='dev')
[2022-02-15 02:29:10,734] {subprocess.py:89} INFO - 02:29:10
[2022-02-15 02:29:10,789] {subprocess.py:89} INFO - 02:29:10  1 of 4 START test accepted_values_stg_orders_status__completed__shipped__returned__placed__return_pending [RUN]
[2022-02-15 02:29:10,799] {subprocess.py:89} INFO - 02:29:10  4 of 4 START test unique_stg_orders_order_id.................................... [RUN]
[2022-02-15 02:29:10,807] {subprocess.py:89} INFO - 02:29:10  3 of 4 START test relationships_stg_orders_customer_id__customer_id__ref_stg_customers_ [RUN]
[2022-02-15 02:29:10,900] {subprocess.py:89} INFO - 02:29:10  2 of 4 START test not_null_stg_orders_order_id.................................. [RUN]
[2022-02-15 02:29:13,733] {subprocess.py:89} INFO - 02:29:13  2 of 4 PASS not_null_stg_orders_order_id........................................ [PASS in 2.83s]
[2022-02-15 02:29:14,760] {subprocess.py:89} INFO - 02:29:14  1 of 4 PASS accepted_values_stg_orders_status__completed__shipped__returned__placed__return_pending [PASS in 3.96s]
[2022-02-15 02:29:14,966] {subprocess.py:89} INFO - 02:29:14  3 of 4 PASS relationships_stg_orders_customer_id__customer_id__ref_stg_customers_ [PASS in 4.16s]
[2022-02-15 02:29:15,091] {subprocess.py:89} INFO - 02:29:15  4 of 4 PASS unique_stg_orders_order_id.......................................... [PASS in 4.28s]
[2022-02-15 02:29:15,182] {subprocess.py:89} INFO - 02:29:15
[2022-02-15 02:29:15,184] {subprocess.py:89} INFO - 02:29:15  Finished running 4 tests in 7.26s.
[2022-02-15 02:29:15,201] {subprocess.py:89} INFO - 02:29:15
[2022-02-15 02:29:15,204] {subprocess.py:89} INFO - 02:29:15  Completed successfully
[2022-02-15 02:29:15,206] {subprocess.py:89} INFO - 02:29:15
[2022-02-15 02:29:15,211] {subprocess.py:89} INFO - 02:29:15  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2022-02-15 02:29:15,731] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 02:29:15,796] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=test.my_new_project.stg_orders, execution_date=20220215T022757, start_date=20220215T022853, end_date=20220215T022915
[2022-02-15 02:29:15,860] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 02:29:15,916] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
